src/crypto/OperationalKeystore.h

```diff
@@ -168,6 +168,28 @@ class OperationalKeystore
      */
     virtual CHIP_ERROR SignWithOpKeypair(FabricIndex fabricIndex, const ByteSpan & message,
                                          Crypto::P256ECDSASignature & outSignature) const = 0;
+
+    /**
+     * @brief Create an ephemeral keypair for use in session establishment.
+     *
+     * The caller must Initialize() the P256Keypair if needed. It is not done by this method.
+     *
+     * NOTE: There may be more than one of these in use concurrently, so implementations have to
+     *       be careful to avoid assuming a single instance.
```

Comment:
Is this stating that implementations must support concurrent allocation of more than one keypair, or that implementations must ensure that no more keypairs are concurrently allocated than the underlying allocator can support?

If the former, why not just state more obviously: the stack will try to allocate more than one, so you better support this.

If the latter, this seems to lack guidance on how implementations would actually achieve enforcing that no more key pairs then they can support will be allocated.

Reply:
The former. Let me rephrase